src/app/applications/components/applications-list/applications-list.scss

```diff
@@ -6,15 +6,32 @@
         font-weight: bolder;
         font-size: 15px;
         color: $argo-color-gray-6;
+        padding-top: 0.25em;
+        padding-bottom: 0.5em;
     }
 
     &__info {
         line-height: 24px;
-        border-left: 1px solid $argo-color-gray-4;
         margin: 1em 0;
     }
 
     &__icons {
         line-height: 24px;
     }
+
+    &__entry {
+        border-left: 5px solid $argo-color-gray-4;
+
+        &.health-Healthy {
```

Comment:
Please follow BEM convention. State selector should look like '<block>__<element>--<state>'. So your selectors should be `&--health-Healthy` etc